tests/pytest/eligibility/test_views.py

```diff
@@ -71,6 +72,39 @@ def model_EnrollmentFlow_with_form_class(mocker, model_EnrollmentFlow):
     return model_EnrollmentFlow
 
 
+@pytest.mark.django_db
+def test_eligibility_logged_in_filtering_flows(mocker, model_TransitAgency, client):
```

Comment:
I think the test name should be something like `test_index_filtering_flows`? 

My thinking is that it makes a request to `routes.ELIGIBILITY_INDEX`/the `index` view, and "logged in" doesn't apply to the `eligibility` app.